lib/node_modules/@stdlib/utils/async/if-then/lib/main.js

```diff
@@ -119,7 +119,7 @@ function ifthenAsync( predicate, x, y, done ) {
 			return done( error );
 		}
 		nargs = arguments.length;
-		args = new Array( nargs );
+		args = Array.from({ length: nargs });
```

Comment:
This is not what we want. This project uses ES5. The lint error message tells you explicitly what you should be doing to resolve this issue.